components/google-cloud/google_cloud_pipeline_components/experimental/remote/gcp_launcher/launcher.py

```diff
@@ -37,8 +38,8 @@ def _parse_args(args):
         prog='Vertex Pipelines service launcher', description=''
     )
     parser.add_argument(
-        "--type",
```

Comment:
The naming convension here is for serverless launcher optimization. you should probably keep it as is. 
We'll have non job types as well.

Reply:
Done. Changed back to type.